stdeb.cfg

```diff
@@ -1,5 +1,6 @@
 [colcon-core]
 No-Python2:
-Depends3: python3-distlib, python3-empy, python3-pytest, python3-pytest-cov, python3-setuptools
+Depends3: python3-distlib, python3-empy, python3-pytest, python3-setuptools
+Recommends3: python3-pytest-cov, python3-pytest-repeat, python3-pytest-rerunfailures
```

Comment:
This is completely outside of my knowledge of stdeb, so take this as a question.  It looks like python3-pytest-repeat is only available in Ubuntu Jammy and forwards, and Debian bookworm and forwards.  Similarly, it looks like python3-pytest-rerunfailures is only available in Ubuntu Hirsute and forwards, and Debian Bullseye and forwards.  Since the Suite has things like `bionic` on the list, will `colcon-core` still be able to be released on e.g. Bionic if we merge this?

Reply:
I couldn't find much documentation on the subject, so I built the package and gave this a try.

On Jammy, all of the packages listed under `Recommends3` were included in the dependency lists as if they were part of `Depends3`.

On both Bionic and Focal, there was an additional category present when attempting installation:
```
Recommended packages:
  python3-pytest-repeat python3-pytest-rerunfailures
```

Those packages were not included in either the install lists, and in all cases, installation completed successfully.

Reply:
All right, that sounds good to me then.  In that case, I'll go ahead and approve, thanks for the extra research.